app/src/main/java/com/droidhats/campuscompass/views/MapFragment.kt

```diff
@@ -278,10 +277,49 @@ class MapFragment : Fragment(), OnMapReadyCallback, GoogleMap.OnMarkerClickListe
 
     //implements methods of interface GoogleMap.GoogleMap.OnPolygonClickListener
     override fun onPolygonClick(p: Polygon) {
+
```

Comment:
why double spacing?

Reply:
Woops, that's where I had my debug code to check if I was making in that function. I removed the debug line but forgot to clear the space. Will fix it right away.